sympy/simplify/ratsimp.py

```diff
@@ -55,7 +55,7 @@ def ratsimpmodprime(expr, G, *gens, quick=True, polynomial=False, **args):
     ==========
 
     .. [1] M. Monagan, R. Pearce, Rational Simplification Modulo a Polynomial
-        Ideal, https://dl.acm.org/doi/pdf/10.1145/1145768.1145809
+        Ideal, https://dl.acm.org/doi/abs/10.1145/1145768.1145809
```

Comment:
"https://doi.org/10.1145/1145768.1145809" is better.

Reply:
Please respond to this review.

Reply:
They seem the same to me, I added the new one because of the linkcheck.
What do you think? Should I put [https://doi.org/10.1145/1145768.1145809](url) instead?

Reply:
The doi link is the one that's immune to link rot. Even if ACM changes their link format the doi link will always continue to redirect to the correct site.